### ISSUE TITLE
ImportError: cannot import name 'run_inference_engine_annotation

### ISSUE THREAD

**Issue by Angel3636** [opencv/cvat#1798]
### My actions before raising this issue
- [yes ] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [ ] Searched [past issues](/issues)

Trying to run the run_model.py on existing bin file.

### Expected Behaviour
 auto annotated images

### Current Behaviour
Error "  File "utils/auto_annotation/run_model.py", line 18, in <module>
    from cvat.apps.auto_annotation.inference import run_inference_engine_annotation
ImportError: cannot import name 'run_inference_engine_annotation' from 'cvat.apps.auto_annotation.inference' (/home/sagarjadhav/software/cvat/utils/auto_annotation/../../cvat/apps/auto_annotation/inference.py)"

### Steps to Reproduce (for bugs)

1. Running the command from cvat root: 
python utils/auto_annotation/run_model.py --py /home//Desktop/temp/ssd_inception_v2_coco_2018_01_28/interp.py --xml /home//Desktop/temp/ssd_inception_v2_coco_2018_01_28/frozen_inference_graph.xml   --bin /home//Desktop/temp/ssd_inception_v2_coco_2018_01_28/frozen_inference_graph.bin --json /home//Desktop/temp/ssd_inception_v2_coco_2018_01_28/mapping.json --image-files /home//Desktop/temp/ssd_inception_v2_coco_2018_01_28/1.jpg /home//Desktop/temp/ssd_inception_v2_coco_2018_01_28/2.jpg /home//Desktop/temp/ssd_inception_v2_coco_2018_01_28/3.jpg --show-images --show-image-delay 2000  

### Context
Planning to create custom model for auto annotation.

### Your Environment

- Git hash commit : commit 2a349d024318c685a85178ad4401100d486e7a65
- Docker version: Running on local machine in development mode
- Operating System and version (e.g. Linux, Windows, MacOS): Linux

**Comment by azhavoro** [comment]
@Angel3636 Thanks for the report!

**Comment by benhoff** [comment]
@Angel3636 , any chance you could test https://github.com/opencv/cvat/pull/1826 and see if it resolves your issue?

**Comment by CLKVoss** [comment]
Hello @benhoff 
I have testet your changes. Now the import works fine. But unfortunately `run_model.py` chrashes with this error:

```bash
Traceback (most recent call last):
  File "cvat/apps/auto_annotation/run_model.py", line 264, in <module>
    main()
  File "cvat/apps/auto_annotation/run_model.py", line 188, in main
    results, _ = runner.run(restricted=restricted)
  File "/home/django/cvat/apps/auto_annotation/inference.py", line 144, in run
    self._add_shapes(processed_detections.get_shapes(), result["shapes"])
  File "/home/django/cvat/apps/auto_annotation/inference.py", line 165, in _add_shapes
    "attributes": _process_attributes(shape["attributes"], label_attr_spec),
  File "/home/django/cvat/apps/auto_annotation/inference.py", line 40, in _process_attributes
    if attr_text in label_attr_spec:
TypeError: argument of type 'NoneType' is not iterable
```

I call `run_model.py` with the keyword arguments `--py, --xml, --bin, --json, --image-files` and checked several times if I made a mistake with theses keyword arguments.

When I run the `run_model.py` script,   `label_attr_spec` is `None` and it crashes as mentioned above.

When I run my model online, i.e. using the cvat web gui, `label_attr_spec` is `{'confidence': 1}` and the annotation works fine.

I used the model which is shown here: https://github.com/opencv/cvat/blob/develop/cvat/apps/auto_annotation/README.md#person-vehicle-bike-detection-crossroad-0078-openvino-toolkit

Maybe you have an idea what is going on and why `label_attr_spec' has different values.

**Comment by benhoff** [comment]
@CLKVoss , thanks for testing! Let me stand up an environment and debug it.

**Comment by benhoff** [comment]
@CLKVoss , how did you apply the code to test the changes?

Because the `attribute_spec` is set explicitly, I'm not sure how it can ever be `NoneType`

See this line:
https://github.com/benhoff/cvat/blob/ad4b151b96344386b6f04fb6e5b6446431a5df40/utils/auto_annotation/run_model.py#L179

Wondering if it was a copy and paste error or if you fetched the code directly from git?

**Comment by CLKVoss** [comment]
@benhoff , I used your file (`run_model.py`) directly from github.

The error still occurs. I just added some debug code in `inference.py`, that is the reason the error message says line 40.
```bash
  File "/home/django/cvat/apps/auto_annotation/inference.py", line 40, in _process_attributes
    if attr_text in label_attr_spec:
```

**Comment by benhoff** [comment]
I think @azhavoro successfully troubleshot the problem that you were having. I have updated the branch with his suggestions.

Would you mind pulling and seeing if the error persists? I'll try to reproduce locally if you don't get a chance to verify that this is fixed.

**Comment by Angel3636** [comment]
> 
> 
> @Angel3636 , any chance you could test #1826 and see if it resolves your issue?

Sorry for the late response. Can you please let me know how do i pull those changes, I have tried "git pull d9dceaf6a8dfc6fa55651ca3e83f0090a9332564" but i got error 
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.

Please let me know how so i pull changes locally?

**Comment by CLKVoss** [comment]
@benhoff Sorry for the late response. I tested the changes @azhavoro mentioned and it seems to work. The `run_model.py` does not throw any error anymore. Thank you for your help.

**Comment by CLKVoss** [comment]
@Angel3636 I just clicked on the button `Browse files` of the commit and downloaded the zip.
![2020-07-08 08_51_40-updated auto annotation runner to use new api · opencv_cvat@d9dceaf](https://user-images.githubusercontent.com/63105087/86887103-5e9d2200-c0f8-11ea-8ff0-2fb1cc51c6c7.png)

**Comment by azhavoro** [comment]
Should be fixed #1767
